packages/adapters/wagmi/src/client.ts

```diff
@@ -217,10 +217,6 @@ export class WagmiAdapter implements ChainAdapter {
   }
 
   public construct(appKit: AppKit, options: AppKitOptions) {
-    if (!options.projectId) {
```

Comment:
Where do we check this now? Wagmi adapter requires projectId to be set unlike the other adapters?

Reply:
Yeah so we don't construct the parameters unless `projectId` has been provided. 

I've implemented a logic [here](https://github.com/reown-com/appkit/blob/feat/debug/packages/appkit/src/client.ts#L424-L430) that does an early return if `projectId` is not provided which prevents adapters to construct.

Reply:
But the thing is, adapter classes and Appkit class are initialized independently. 

For example for ethers example, we don't already pass projectId to EthersAdapter class, we need to pass it only to AppKit.
Bot for wagmi example, they need to pass projectId to WagmiAdapter as well: https://github.com/reown-com/appkit/blob/main/apps/laboratory/src/pages/library/wagmi.tsx#L16

Reply:
> if projectId is not provided which prevents adapters to construct

Even though we pass to the AppKit, there is possibility for users that they might forget passing projectId to WagmiAdapter still

Reply:
Makes sense. I pushed a change [here](https://github.com/reown-com/appkit/pull/2929/commits/69e649c1ac3f4ba2a66bf855d9d59ad0dc96d1a4) 🙏